autoload/airline/parts.vim

```diff
@@ -82,7 +82,7 @@ function! airline#parts#readonly()
 endfunction
 
 function! airline#parts#filetype()
-  return winwidth(0) < 100 && strlen(&filetype) > 3 ? matchstr(&filetype, '...'). '…' : &filetype
+  return winwidth(0) < 100 && strlen(&filetype) > 3 ? matchstr(&filetype, '...'). iconv(nr2char(0x2026, 1), "utf-8", &encoding) : &filetype
 endfunction
```

Comment:
instead of an expansive iconv() call (which might not be available in your vim, rather do something like [this](https://github.com/vim-airline/vim-airline/blob/master/autoload/airline/extensions/branch.vim#L162)